### DISCUSSION TITLE
Single message threads should not force you to open the thread to mark it as read

### DISCUSSION THREAD

**Discussion by ara4n** [element-hq/element-meta#900] [Product Feedback]
It's quite common for someone to start a thread with a single reply, which then never gets a response. Cureently this causes a stuck unread notif unless you explicitly open the thread to read the msg. This is suboptimal, because the message is presented inline in the main timeline anyway, so from the user's point of view they have already read it. "Why do i have to click into this thread to make the notif go away? I've already read it!"

**Comment by daniellekirkwood** [comment]
I'm going to move this to a discussion as it's an Enhancement to the Threads feature, and we don't know the size of this use case... The main timeline and the thread panel only show part of the message - so this use case only stands for those one off messages that are smaller. 

Examples: 
The thread panel:
<img width="779" alt="Screenshot 2023-01-04 at 19 50 21" src="https://user-images.githubusercontent.com/89144281/210637752-7c04449f-f701-4aea-8238-258dcb0bf24d.png">

The main timeline:
<img width="633" alt="Screenshot 2023-01-04 at 19 50 08" src="https://user-images.githubusercontent.com/89144281/210637812-db85d4fe-ef86-44a1-b6d1-b4ce24feea29.png">
